### PR TITLE
refactor slash command GuildCommand.ts #331

### PR DESCRIPTION
**Pull request by romain22222**

### CODE REVIEW

File: src/core/Tools.js [hunk 1]
```diff
@@ -127,21 +133,14 @@ global.giveItem = async (entity, item, language, discordUser, channel, resaleMul
  * @return {Promise<*>}
  */
 global.destroyPotionMessage = async (channel, language, discordUser, item, isAutoSell = false) => {
-	const titleEmbedDestroyPotionMessage = isAutoSell
-		? JsonReader.commands.sell.getTranslation(language).soldMessageAlreadyOwnTitle
-		: JsonReader.commands.sell.getTranslation(language).potionDestroyedTitle;
-	return await channel.send({ embeds: [
-		new DraftBotEmbed()
+	const titleEmbedDestroyPotionMessage = isAutoSell ? JsonReader.commands.sell.getTranslation(language).soldMessageAlreadyOwnTitle : JsonReader.commands.sell.getTranslation(language).potionDestroyedTitle;
```
Comment (romain22222) [inline src/core/Tools.js:136]:
```suggestion
	const titleEmbedDestroyPotionMessage = isAutoSell ?
	JsonReader.commands.sell.getTranslation(language).soldMessageAlreadyOwnTitle 
	: JsonReader.commands.sell.getTranslation(language).potionDestroyedTitle;
```

File: src/commands/guild/GuildCommand.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+			progressBar(1, 1)
+		);
+	}
+	await interaction.reply({embeds: [embed]});
+}
+
+export const commandInfo: ICommand = {
+	slashCommandBuilder: new SlashCommandBuilder()
+		.setName("guild")
+		.setDescription("Displays the guild via a player or via its name")
```
Comment (BastLast) [inline src/commands/guild/GuildCommand.ts:150]: through

File: src/commands/guild/GuildCommand.ts [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+		.addUserOption(option => option.setName("user")
+			.setDescription("The user you want to see the inventory")
```
Comment (BastLast) [inline src/commands/guild/GuildCommand.ts:152]: the inventory ?

File: src/commands/guild/GuildCommand.ts [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+			.setRequired(false)
+		)
+		.addStringOption(option => option.setName("guild_s_name")
+			.setDescription("The guild's name to display")
```
Comment (BastLast) [inline src/commands/guild/GuildCommand.ts:156]: tu display le name ou la guilde ?

File: src/commands/guild/GuildCommand.ts [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+			.setRequired(false)
+		)
+		.addNumberOption(option => option.setName("rank")
+			.setDescription("The rank of the player you want to see the inventory")
```
Comment (BastLast) [inline src/commands/guild/GuildCommand.ts:160]: l'inventory ?

**Review by BastLast — CHANGES_REQUESTED** [review]

**Review by BastLast — APPROVED** [review]

**Pull request merged**